metals/src/main/scala/scala/meta/internal/metals/doctor/Doctor.scala

```diff
@@ -45,30 +45,12 @@ import org.eclipse.{lsp4j => l}
  * At the moment, the doctor only validates that SemanticDB is enabled for all projects.
  */
 final class Doctor(
-    workspace: AbsolutePath,
-    buildTargets: BuildTargets,
-    diagnostics: Diagnostics,
+    foldersParams: List[DoctorFolderParams],
```

Comment:
@tgodzik, I assume we want to have one global doctor, right?

Reply:
There are two solutions here:
1. Separate doctor per workspace folder
2. Join all the doctor outputs in the webview

I would suggest 1., but 2. should also be ok, though more work.